### ISSUE TITLE
using of FID

### ISSUE THREAD

**Issue by zzgg1** [tianxintao/Online-Motion-Style-Transfer#2]
Thanks for your amazing code! I am wondering if it could be possible that you provide pretrained model for FID, and elaborate on the detailed training procedures of FID

**Comment by tianxintao** [comment]
Most of the training code should be available here: [fid.py](https://github.com/tianxintao/Online-Motion-Style-Transfer/blob/main/fid.py)
Basically, a denoising autoencoder is first trained with reconstruction loss, which is then used as features for FID. The computation of FID follows the standard FID formula.
